cmake/unix.cmake

```diff
@@ -18,7 +18,7 @@ if(NOT IOS AND NOT ANDROID AND NOT WIN32)
     set(CURL_CA_PATH none)
     message(STATUS "using local curl: ${LIBCURL_ROOT}")
     add_subdirectory(${LIBCURL_ROOT})
-    set(CURL_INCLUDE_DIRS ${LIBCURL_ROOT}/include)
+    set(CURL_INCLUDE_DIR ${LIBCURL_ROOT}/include)
```

Comment:
This should stay as plural because `FindCURL` sets it CURL_INCLUDE_DIRS (and if it comes that way it can potentially contain multiple dirs, plus agrees with how other cmake library finding code handles things).